stage0/run.go

```diff
@@ -167,6 +167,13 @@ func generatePodManifest(cfg PrepareConfig, dir string) ([]byte, error) {
 		}
 
 		if execOverride := app.Exec; execOverride != "" {
+			// Create a minimal App section if not present
+			if am.App == nil {
+				ra.App = &types.App{
+					User:  "0",
```

Comment:
I originally implemented it that way but then I thought that running `rkt` as a non-root in the near future is very unlikely. I can re-add it, the result would be the same and we'll be prepared.